src/server/rpc/procedures/roboscape/ciphers.js

```diff
@@ -1,4 +1,4 @@
-const Speck32 = require('./speck.js');
+const Speck32 = require('./speckjs/src/ciphers/speckNative.js');
```

Comment:
I think this file was included by accident

Reply:
(Same with speck.js)

Reply:
This can be fixed with `git checkout master -- path/to/file/to/remove/from/commit` then commit the changed file. I just had to look this up myself so I thought I would post it here :)